Application/Dto/PortDTO.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace Application.Dto
+{
+    public class PortDTO
+    {
+        [Required]
+        public int Id { get; set; }
+        [Required]
+        public string Name { get; set; }
+        [Required]
+        public string Lat { get; set; }
```

Comment:
Why string type is preferred for longitude and latitude?

Reply:
In GIS coordinates usualy are not manipulated so string is easier to use.